src/lib.rs

```diff
@@ -192,3 +196,40 @@ pub fn ec_private_keys(
         _ => None,
     })
 }
+
+/// Return the first public key found in `rd`.
+///
+/// Yields the first PEM section describing a public key, or an error if a
+/// problem occurs while trying to read PEM sections.
+#[cfg(feature = "std")]
+pub fn public_key(
```

Comment:
I feel like we don't need these convenience helper methods for every type of `Item`, and since this feels kind of niche I think we might leave them out here.

Reply:
Just to be sure, do you mean that both helper functions are unnecessary or just the `public_key()` one?

Reply:
I would drop `public_key()` but keep `public_keys()`. Generally I would expect anything that is configuring a peer's public keys to work when given a file with multiple public keys in it.